source/server/connection_handler_impl.cc

```diff
@@ -187,7 +187,7 @@ void ConnectionHandlerImpl::ActiveSocket::continueFilterChain(bool success) {
 
 void ConnectionHandlerImpl::ActiveListener::onAccept(
     Network::ConnectionSocketPtr&& socket, bool hand_off_restored_destination_connections) {
-  Network::Address::InstanceConstSharedPtr local_address = socket->localAddress();
+  // Network::Address::InstanceConstSharedPtr local_address = socket->localAddress();
```

Comment:
Feel free to drop that line altogether: however, I suggest doing that in a separate PR as it's an unrelated change.

Reply:
Yes, please revert this and do another PR if this line is dead. Thanks!

/wait

Reply:
OK, understood.